controller/appcontroller.go

```diff
@@ -72,6 +72,9 @@ const (
 	orphanedIndex = "orphaned"
 )
 
+// refreshAfterForDependencies defines the interval for refresh while waiting for dependency application
+var refreshAfterForDependencies time.Duration = 2 * time.Second
```

Comment:
Move this to constant as the value is not getting updated anywhere

Reply:
I moved it to the const block for now, along with a TODO comment  that the duration probably should be configurable in the future.